cli.py

```diff
@@ -24,3 +24,11 @@
     help = 'Censys API secret. Can also be defined using the CENSYS_API_SECRET environment variable',
     dest = 'censys_api_secret'
 )
+
+parser.add_argument(
+    '--non-commercial',
+    help = 'For Censys non-commercial account, results should be limited',
```

Comment:
How about:

`Limit search results to 1000 items for Censys non-commercial accounts`